lib/gym/generators/build_command_generator.rb

```diff
@@ -68,7 +68,9 @@ def pipe
 
       def xcodebuild_log_path
         file_name = "#{Gym.project.app_name}-#{Gym.config[:scheme]}.log"
-        containing = File.expand_path("~/Library/Logs/gym")
+        file_dir = Gym.config[:buildlog_path]
+        file_dir ||= "~/Library/Logs/gym"
+        containing = File.expand_path(file_dir)
```

Comment:
simplify to `containing = File.expand_path(Gym.config[:buildlog_path)`
and set the default value in `options.rb`